composer.json

```diff
@@ -12,7 +12,7 @@
 	"prefer-stable": true,
 	"require": {
 		"php": "^7.4 || ^8.0",
-		"phpstan/phpstan": "^1.10.9",
+		"phpstan/phpstan": "^1.10.9 || ^1.11.0",
```

Comment:
`^1.10.9` already includes 1.11.*, while `~1.10.9` would not. Or am I missing something here?

Reply:
Nope, your're right. My fault. 
I'm just thinking about how to handle the package version now if it's compatible with 1.10 and 1.11. The package version was always not based on semantic versioning but the PHPStan version it is based on. But here was no breaking change form 1.10 to 1.11 this time. So IMO there is no need to change the package version but it looks now "outdated" if we keep 1.10. It just feels weird...